integration_tests/Cargo.toml

```diff
@@ -646,13 +650,14 @@ path = "../rusoto/services/ram"
 
 [dev-dependencies]
 env_logger = "0.7"
-futures = "0.1.16"
-futures-fs = "0.0.3"
+bytes = "0.5"
+futures = "0.3"
 log = "0.4.1"
 rand = "^0.7.2"
 time = "0.1.37"
-reqwest = "0.9"
-http = "0.1.17"
+reqwest = { git = "https://github.com/seanmonstar/reqwest.git", branch = "master", features = ["json"] }
```

Comment:
reqwest also released a version with async await support... shouldn't it be used instead of using master branch directly?

Reply:
Yep, needs to be updated to be 0.10.